src/middleware/express/make-http-request.ts

```diff
@@ -18,14 +18,18 @@ import * as http from 'http';
 
 import {StackdriverHttpRequest} from '../../http-request';
 
+export interface ServerRequest extends http.IncomingMessage {
+  originalUrl: string;
+}
+
 export function makeHttpRequestData(
-  req: http.IncomingMessage,
+  req: ServerRequest,
```

Comment:
strictly speaking this change is breaking for existing typescript users. However, given that the middleware is marked experimental, this is okay to ship as a non-major.